lib/fork.js

```diff
@@ -5,18 +5,19 @@ var Promise = require('bluebird');
 var debug = require('debug')('ava');
 var send = require('./send');
 
-module.exports = function (args) {
+module.exports = function (args, options) {
 	if (!Array.isArray(args)) {
 		args = [args];
 	}
+	if (!options || typeof options !== 'object') {
```

Comment:
It should use https://github.com/sindresorhus/object-assign so not to mutate the user-supplied object.